### DISCUSSION TITLE
Adding an impact category

### DISCUSSION THREAD

**Discussion by mwskelton** [LCA-ActivityBrowser/activity-browser#645] [Q&A]
I am curious whether there is documentation on how a new impact category can be added to the activity browser (or the underlying BW2). I see it is possible to duplicate an impact category already present, but the only obvious thing that can be modified in the copy is then the uncertainty values. Any tips would be appreciated!

**Comment by marc-vdm** [comment]
Sorry for the extremely delayed response. Adding impact categories is currently not possible in AB. You can read more [here](https://docs.brightway.dev/en/legacy/content/api/bw2data/method/index.html)  about adding impact categories in Brightway, you can ask for more help there too!
